tool/mkconfig.rb

```diff
@@ -229,6 +229,15 @@ module RbConfig
   print "  CONFIG[#{v.dump}] = #{(versions[v]||vars[v]).dump}\n"
 end
 
+# Get the ABI version
+File.foreach(File.join(srcdir, "include/ruby/internal/abi.h")) do |l|
+  m = /^\s*#\s*define\s+RUBY_ABI_VERSION\s+(\d+)/.match(l)
+  if m
+    print "  CONFIG[\"ruby_abi_version\"] = \"#{m[1]}\"\n"
```

Comment:
Do we still want to define a new `ruby_abi_version` key in `RbConfig`? Isn't reusing `RbConfig::CONFIG['ruby_version']` much better? (that's already the ABI, used by RubyGems, Bundler, etc, and makes it much more integrated)

Reply:
I opened #5591.